paper.tex

```diff
@@ -805,27 +809,45 @@ \section*{Inheritance and ancestral material}\label{Ancestry_resolution}
-simulation algorithm
+ancestral to the sample. For each node, we also keep track of the number of
+descendant samples over each genomic interval, and stop tracking that
+interval once the lineages from all samples have coalesced. For example,
+the edge from \textsf{k} to \textsf{n} omits the region $(0,2]$ because it has
+already fully coalesced (indicated by a fully black genomic region).
+This process is directly analogous to Hudson's simulation algorithm
 (Appendix XXX).
 
-These alternatives of recording the direct inheritance of genetic material from
-parent to child vs the propagation of genomic intervals ancestral to the sample
-are analogous to running forwards-in-time vs backwards-in-time simulations.
-Indeed, the process of taking a graph with edges annotated with direct parent-child
-inheritance (Fig~\ref{fig-ancestry-resolution}B) and producing the equivalent
-``ancestry resolved'' graph in which only those genomic intervals that are
-ancestral to the sample are retained (Fig~\ref{fig-ancestry-resolution}C)
-is the same key algorithm underlying recent advances in forwards-in-time simulation.
+In practice, the backwards-in-time Griffiths process that produces
+ARGs such as that shown in Figs~{fig-ancestry-resolution}A~and~B
+is rarely simulated, because the resulting ``big ARG'' (Appendix XXX) is
+dominated by nodes such as \textsf{g} and \textsf{j} which contain information
+about inheritance in non-ancestral material, as well as nodes such as
+\textsf{o} and \textsf{q} which track details of ancestry even once the
+local genomic region has fully coalesced. Instead, simulating
+ARGs from the coalescent with recombination is usually done by
+running Hudson's algorithm to produce a sample-resolved ARG directly.
+Moreover, when inferring ARGs from genetic data, there is no information
+in the sample set that can be used to resolve these nodes, so that they
+are essentially unknowable.
+
+Nevertheless, there is a practical use for converting a direct-inheritance
+gARG to a sample resolved version. It is the key algorithm underlying recent
+advances in forwards-in-time simulation.
 \cite{kelleher2018efficient} developed a method in which forwards-in-time
 simulations are performed by recording all parent-child genetic relationships
 (i.e.~annotating the edges in the simulated pedigree with the genomic
 intervals inherited by a child from a parent) and periodically ``simplifying''
 this structure to retain only ancestral relationships that are relevant
 to the ancestry of the current population. In many situations, using this approach to
 record the full genetic ancestry of the current population is orders of magnitude faster
-than the alternative of tracking
-sequences~\citep{kelleher2018efficient,haller2018tree}.
+than the alternative of tracking neutral genetic variation directly~\citep{kelleher2018efficient,haller2018tree}.
+
+It is worth noting that nodes \textsf{g},  \textsf{j}, \textsf{o} and \textsf{q}
```

Comment:
We should probably not ask the reader to look at the figures so much, or expect them to keep them in their head. I've been trying to bring references to specific things in figures down to a bare minimum, with concrete discussions kept to the captions.

I think we can say this without referring to specific nodes, right?

Reply:
Yeah, good point. We're quite a long way from the figure now. I'll rephrase.